spark2/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -388,6 +392,40 @@ public boolean isVectorizationEnabled(FileFormat fileFormat) {
     }
   }
 
+  private int batchSize(boolean isParquetOnly, boolean isOrcOnly) {
+    if (isParquetOnly) {
+      return batchSize(FileFormat.PARQUET);
+    } else if (isOrcOnly) {
+      return batchSize(FileFormat.ORC);
+    } else {
+      return 0;
+    }
+  }
+
+  private int batchSize(FileFormat fileFormat) {
```

Comment:
This PR is a temporary solution until we consume #3132 that will put this logic in one place for Spark 2 and 3.